workspaces/tech-insights/plugins/tech-insights/src/components/ScorecardsPage/ScorecardsPage.tsx

```diff
@@ -52,7 +52,9 @@ export const ScorecardsPage = () => {
 
   const { value, loading, error } = useAsync(async () => {
     const checks = await api.getAllChecks();
-    const result = await api.runBulkChecks([], filterSelectedChecks);
+    const result = (await api.runBulkChecks([], filterSelectedChecks)).filter(
```

Comment:
Is this going to be confusing to not see these missing entities? Could it make sense to display something like 'no result' for these cases?

Reply:
The problem is that if you have a lot of entities, these will be shown without result and therefore hiding other entities with results, which are on an other page.

I could imagine two other approaches to the problem:
- sort entities in the table acording to the results
- Add a filter to show/hide entities with or without results

I think it would make sense to use a filter on the ui, @Xantier what do you think?

Reply:
Yeah, that sounds like a good approach to me. That would leave the availability of all entities intact but would reduce the visual clutter if wanted.

Reply:
Nice, will refactor it to a filter.